bootloader/src/pyi_win32_utils.c

```diff
@@ -444,20 +493,29 @@ pyi_win32_utf8_to_mbs(char * dst, const char * src, size_t max)
 int
 pyi_win32_mkdir(const wchar_t *path)
 {
-    wchar_t stringSecurityDesc[] = // ACE String :
-        L"D:" // DACL (D) :
-        L"(A;" // Authorize (A)
-        L";FA;" // FILE_ALL_ACCESS (FA)
-        L";;S-1-3-4)"; // For the current directory owner (SID: S-1-3-4)
+    char *sid = pyi_win32_get_user_sid();
+    char stringSecurityDesc[PATH_MAX];
+
+    // ACE String :
```

Comment:
Please  don't hide the call in the variable declaration
```suggestion
    char *sid;
    char stringSecurityDesc[PATH_MAX];

    // ACE String :
    sid = pyi_win32_get_user_sid();
```

Reply:
Fixed.